### DISCUSSION TITLE
How to create a project for a text file?

### DISCUSSION THREAD

**Discussion by Roxanne0330** [inception-project/inception#4097] [Q&A]
I try to import a text file, but it shows "Error importing project: IOException: Invalid ZIP file". Do you know how to solve the problem? Thank you!

**Comment by reckart** [comment]
To import a text file, first create a new project. E.g. click on the "Create new project..." button or on the small arrow besides it and select e.g. "Basic annotation".

<img width="557" alt="Screenshot 2023-07-14 at 07 11 49" src="https://github.com/inception-project/inception/assets/1410238/41c4d048-f9a9-42d2-ab83-440d0985ace2">

Once the project has been created, go do the **Settings** panel and then to the **Documents** panel. Here you can upload your documents.
